### ISSUE TITLE
Deployment: "Our recommended PaaS services" should match the one in the NodeJS section

### ISSUE THREAD

**Issue by DrantDumani** [TheOdinProject/curriculum#28862]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

The [Our recommended PaaS services segment in the NodeJS path](https://www.theodinproject.com/lessons/node-path-nodejs-deployment#our-recommended-paas-services) has received several updates over time to remove options that have no longer offer free services (such as Heroku and Fly.io). The lesson has also been updated to include more options that can be used to host databases for free. 

The [Ruby on Rails version of the lesson](https://www.theodinproject.com/lessons/ruby-on-rails-deployment#debugging-and-troubleshooting-deployments) still mentions Heroku and Fly.io. It also lacks the aforementioned database options that the NodeJS path lesson talks about.

As a proposed solution, the "Our recommended PaaS services" segment should be updated to match that of the NodeJS path.

### Path

Ruby / Rails

### Lesson Url

https://www.theodinproject.com/lessons/ruby-on-rails-deployment

### (Optional) Discord Name

Elemeandor

**Comment by KevinMulhern** [comment]
Thanks for the suggestion @DrantDumani. I agree that Koyeb, and Neon would be worth including, both of those have seem to have decent guides for Rails. 

But I'd be hesitant to remove Fly.io and Heroku from the options. The intention of this lesson wasn't to only include options with free plans (although that is a big plus), but what services were easiest to get up and running with. I think both Heroku and Fly still probably have the best DX out of all the available PaaS options available.

**Comment by MaoShizhong** [comment]
Related: #28491

Fly and Adaptable (as well as Heroku at the time of milestone release) were removed from the Node version because we weren't comfortable with the idea of recommending paid-only services in the section following the wording:

> Choosing a PaaS provider was once a simple decision. Heroku had a free tier that gave you everything needed to host as many small apps as you wanted, but they unfortunately discontinued it in 2022.
>
> Luckily, there are still plenty of other great options out there. The downside is that they all have very limited free tiers. For this reason, and to accommodate as many of our learners as possible, we’re going to recommend a range of options instead of just one.

But that was for the Node side only. I didn't want to make the same decision for the Rails side of things as well.

**Comment by JoshDevHub** [comment]
@DrantDumani Do you still have an interest in working on this?

I agree with Kevin that I don't think I'd want Fly or (especially) Heroku removed from the Rails deployment guide, but I do think some of the ones that have been added to the Node deployment guide could be good to have. Would definitely welcome a PR that added Koyeb and Neon as options for learners to use. Let me know if you want to work on it.

**Comment by DrantDumani** [comment]
@JoshDevHub yeah, I'd be down to add those options to the Rails deployment lesson.
